src/Contexts/CartContext.js

```diff
@@ -0,0 +1,35 @@
+import React ,{createContext,useContext,useReducer} from "react"
+
+
+const CartContext = createContext()
+
+function reducerfn(previousState,action)
+{
+    switch(action.type)
+    {
+        case "ADD_TO_CART":
+            return {...previousState,data:[...previousState.data,...action.payload.data]}
+        case "REMOVE_FROM_CART":
+            return {...previousState}
+        
+        case "ADD_TO_WISHLIST":
+            return {...previousState,wishList:[...previousState.wishList,...action.payload.data]}
+        
+        default:
+            return {...previousState}
+    }
+}
+
+
+function useCartContext()
+{
+    const {state,dispatch} = useContext(CartContext)
+    return {state,dispatch}
+}
+function CartContextProvider({children})
+{
+    const [state,dispatch] = useReducer(reducerfn,{data:[],wishList:[]})
```

Comment:
I might wrong but giving a function name with '-' hiphen may not be good, camelCase is good practice

Reply:
Thank you for the suggestion changed it